tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,19 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
-    item = "item to remove"
+    item = "c"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
+    vendor_length = len(vendor.inventory)
+    
+    assert result == "c"
```

Comment:
This test should look to see if your function returns `False` if it looks to remove an item that is not in the inventory of a given vendor. How could we change it to implement that logic?